middlewares/x-powered-by/index.ts

```diff
@@ -1,25 +1,13 @@
 import { IncomingMessage, ServerResponse } from "http";
 
-export interface XPoweredByOptions {
-  setTo?: string;
-}
-
-function getUpdaterFn(
-  value: string | undefined
-): (res: ServerResponse) => void {
-  if (value) {
-    return (res) => {
-      res.setHeader("X-Powered-By", value);
-    };
-  } else {
-    return (res) => {
-      res.removeHeader("X-Powered-By");
-    };
-  }
+function getUpdaterFn(): (res: ServerResponse) => void {
+  return (res) => {
+    res.removeHeader("X-Powered-By");
+  };
 }
 
-function xPoweredBy(options: Readonly<XPoweredByOptions> = {}) {
-  const updateResponse = getUpdaterFn(options.setTo);
+function xPoweredBy() {
+  const updateResponse = getUpdaterFn();
```

Comment:
We don't need a fancy updater function any more—we can just return the middleware. For example:

```ts
function xPoweredByMiddleware(
  _req: IncomingMessage,
  res: ServerResponse,
  next: () => void
) {
  res.removeHeader("X-Powered-By");
  next();
}

function xPoweredBy() {
  return xPoweredByMiddleware;
}
```

Could you update so that it looks like that?